packages/preview-middleware/src/base/flp.ts

```diff
@@ -427,16 +431,33 @@ export class FlpSandbox {
                     this.logger
                 );
                 if (success) {
-                    res.status(200).send(message);
+                    this.sendResponse(res, 'text/plain', 200, message ?? '');
                 } else {
-                    res.status(400).send('INVALID_DATA');
+                    this.sendResponse(res, 'text/plain', 400, 'INVALID_DATA');
                 }
             } catch (error) {
-                res.status(500).send(error.message);
+                this.sendResponse(res, 'text/plain', 500, error.message);
             }
         }) as RequestHandler);
     }
 
+    /**
+     * Send a response with the given content type, status and body.
```

Comment:
@heimwege could you enhance the comment and mention that this function is written to work with both `express` and `connect` otherwise, someone might "optimize" it in the future.

Reply:
Done. And the integrated type check will prevent using methods that are not common 🐱